deepfence_server/pkg/integration/splunk/splunk.go

```diff
@@ -23,64 +31,93 @@ func New(ctx context.Context, b []byte) (Splunk, error) {
 }
 
 func (s Splunk) SendNotification(ctx context.Context, message string, extras map[string]interface{}) error {
+	s.client = utils.GetInsecureHttpClient()
 	var msg []map[string]interface{}
 	d := json.NewDecoder(strings.NewReader(message))
 	if err := d.Decode(&msg); err != nil {
 		fmt.Println("Failed to unmarshal message for splunk", err)
 		return err
 	}
+
+	numRoutines := (len(msg) / 10)
+	if numRoutines == 0 {
+		numRoutines = 1
+	} else if numRoutines > 10 {
+		numRoutines = 10
+	}
```

Comment:
Could simplify with a one-liner:
```suggestion
   numRoutines := max(min(len(msg)/10, 1),10)
```